### PR TITLE
Add openjpeg port

### PR DESCRIPTION
**Pull request by traversaro**
Related issue : https://github.com/Microsoft/vcpkg/issues/3 .

### CODE REVIEW

File: ports/openjpeg/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+)
+
+vcpkg_install_cmake()
+
+file(REMOVE_RECURSE ${CURRENT_PACKAGES_DIR}/debug/include)
+
+# Handle debug cmake config files (see https://github.com/Microsoft/vcpkg/issues/77)
+file(READ ${CURRENT_PACKAGES_DIR}/debug/lib/openjpeg-2.1/OpenJPEGTargets-debug.cmake OPENJPEG_DEBUG_MODULE)
+string(REPLACE "\${_IMPORT_PREFIX}" "\${_IMPORT_PREFIX}/debug" OPENJPEG_DEBUG_MODULE "${OPENJPEG_DEBUG_MODULE}")
+file(WRITE ${CURRENT_PACKAGES_DIR}/lib/openjpeg-2.1/OpenJPEGTargets-debug.cmake  "${OPENJPEG_DEBUG_MODULE}")
```
Comment (Orvid) [inline ports/openjpeg/portfile.cmake]: This should be getting written to share, not lib.

**Comment by traversaro** [comment]
While working on https://github.com/Microsoft/vcpkg/pull/208#discussion_r84783579 , I noticed that the static build are not working properly. Changing the PR title to `[WIP]` until we figure how to fix the static builds.

File: ports/openjpeg/portfile.cmake (hunk 1 above)
Comment (ras0219-msft) [inline ports/openjpeg/portfile.cmake]: Thanks for the ping! We should improve the checks here; let's use #77 to track it.

**Comment by traversaro** [comment]
Spurious redistributable runtime dll removed in https://github.com/Microsoft/vcpkg/pull/208/commits/7658d318366ac149285617d41cf97d63a2cb9adc , comments in https://github.com/Microsoft/vcpkg/pull/208#issuecomment-255866299 resolved.

**Comment by alexkaratarakis** [comment]
Looks good, thanks!
I only added a call to vcpkg_copy_pdbs().

**Pull request merged**